src/applications/disability-benefits/all-claims/tests/pages/uploadForm4192.unit.spec.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import {
+  DefinitionTester, // selectCheckbox
+} from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../../config/form.js';
+import { ERR_MSG_CSS_CLASS } from '../../constants';
+
+describe('4192 record upload', () => {
+  const page = formConfig.chapters.disabilities.pages.uploadForm4192;
+  const { schema, uiSchema, arrayPath } = page;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        arrayPath={arrayPath}
+        pagePerItemIndex={0}
+        definitions={formConfig.defaultDefinitions}
+        schema={schema}
+        data={{
```

Comment:
Can we change the data to reflect the `8940/4192` workflow? Or do we need this?

Reply:
You are right, this isn't necessary